backup_and_restore/application_backup_and_restore/oadp-advanced-topics.adoc

```diff
@@ -35,4 +41,6 @@ For more information about OADP Data Mover, see xref:../../backup_and_restore/ap
 
 For more information about using Restic with OADP, see xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#backing-up-applications[Backing up applications with File System Backup: Kopia or Restic].
 
+For more information about MTC converting storage classes see xref:../../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-converting-storage-classes_advanced-migration-options-mtc[Migration converting storage classes].
```

Comment:
Similar to Mike's comment above, this is not compliant with Additional resources formatting but I see it's based on existing content. This one is a pretty easy fix though, and directly impacts how things render on the portal; I think it's worth fixing before merge.

To make this compliant, convert these four items to a bulleted list, and remove all text that's not in the link text, like so:

```
* xref:../../backup_and_restore/application_backup_and_restore/oadp-advanced-topics.adoc#oadp-different-kubernetes-api-versions[Working with different Kubernetes API versions on the same cluster]
* xref:../../backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc#backing-up-applications[Using Data Mover for CSI snapshots]
* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#backing-up-applications[Backing up applications with File System Backup: Kopia or Restic]
* xref:../../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-converting-storage-classes_advanced-migration-options-mtc[Migration converting storage classes]
```

Reply:
@jeana-redhat 
Thank you for your suggestion.